README.md

```diff
@@ -150,15 +150,15 @@ ruby mai.rb
 
 ## 👥 Authors <a name="authors"></a>
 
-> Mention all of the collaborators of this project.
 
-👤 **Author1**
+
+👤 **Mohamed Eldeeb**
 
 - [Mohamed Eldeeb](https://github.com/eng-mohamed-eldeeb)
 
 
 
-👤 **Author2**
+👤 **Mauricio Mendoza**
 
 - [R. Mauricio Mendoza](https://github.com/rbreva) 
 
```

Comment:
- [ ]  You should add command to run test in of your application. Your readme documentation should contain all required steps for the user to run your application, and it should the same on tests as well. just add the command user should run to run the test of your application.